### ISSUE TITLE
Update orgs list to make more compelling as showcase

### ISSUE THREAD

**Issue by nassarhayat** [radicle-dev/radicle-interface#89]
We currently have a list of organisations on the [Orgs page](https://app.radicle.network/orgs/) that show an avatar and name/address. 

I am planning to focus projects around the output of setting an org and org node being a listing on this page, so it may be worth optimising and making it a little more substantial. Immediate ideas for things I would like to see:

1. Orgs page moved to home page with sidebar moved to top. e.g. like [snapshot](https://snapshot.org/#/)
2. Sorting them to put most compelling ones at the top. 
3. number of repos anchored in each org
4. I see that there is already an issue for adding the ability to star. Would be good to have a star count here as well.

**Comment by sebastinez** [comment]
Hey @nassarhayat thank you for your issue!

I think there are some great ideas, I would like if you could elaborate a little bit on number 2.
- Which criteria would you recognize to define if a org is more compelling than another one?

**Comment by cloudhead** [comment]
One thing we should do is show the orgs with avatar + name first.

I'm wondering if it makes sense to add ENS support to our subgraph, so that we can fetch everything at once and even filter out orgs that don't have any metadata or anchors.

**Comment by sebastinez** [comment]
> I'm wondering if it makes sense to add ENS support to our subgraph, so that we can fetch everything at once and even filter out orgs that don't have any metadata or anchors.

This sounds great, we could improve the loading time of the orgs time tremendously, is this something that can be set in the subgraph?

**Comment by nassarhayat** [comment]
Unsure on what we should sort by. has Avatar, has name, and number of product for now?

**Comment by cloudhead** [comment]
> > I'm wondering if it makes sense to add ENS support to our subgraph, so that we can fetch everything at once and even filter out orgs that don't have any metadata or anchors.
> 
> This sounds great, we could improve the loading time of the orgs time tremendously, is this something that can be set in the subgraph?

We'd have to track events from the registrar contract, as well as the `NameChanged` event in the org maybe.

**Comment by nassarhayat** [comment]
![image](https://user-images.githubusercontent.com/2205084/135272139-12a63ab9-4380-44df-baa3-b58b9e9fc8d6.png)

good example of effect we want to have on users. number of contributors + number of projects/repos should act as a credibility signal that projects are keen to maintain/grow.

**Comment by cloudhead** [comment]
Yeah I like that. Needs a bit of work on the backend if we don't want to have to fire a bunch more queries.

Alternatively we just keep a short list of the top projects which we curate manually and we can then afford to be inefficient. I think this may yield a better result in the mid term anyway, until the list grows too large to maintain.
